tests/components/vizio/test_config_flow.py

```diff
@@ -739,7 +739,15 @@ async def test_zeroconf_flow(
 
     # Apply discovery updates to entry to mimic when user hits submit without changing
     # defaults which were set from discovery parameters
-    user_input = result["data_schema"](discovery_info)
+    user_input = result["data_schema"](
```

Comment:
I am not sure if this is the best way to rework the tests now that discovery_info no longer get "hijacked" in the config flow.